bears/python/PycodestyleBear.py

```diff
@@ -0,0 +1,48 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+from coalib.bears.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='pycodestyle',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+) (?P<column>\d+) (?P<message>.*)')
+class PycodestyleBear:
+    """
+    A wrapper for the tool ``pycodestyle`` formerly known as ``pep8``.
+    """
+    LANGUAGES = {"Python", "Python 2", "Python 3"}
```

Comment:
I could not find anything that states this explicitly. But I believe it does not matter, because `pycodestyle` only checks the style conventions of PEP8, which apply to both versions. Syntax errors are **not** detected, e.g. a print statement yields no error if run with python3. I briefly scanned the source code and they do at some point query the python version, but it's for some file encoding related issue. So I think were good.